### PR TITLE
Provides the initial version of the gitops module to create the instance

### LINKED ISSUE #2: Create a gitops module to provision an event streams instance

### PR DESCRIPTION
**Pull request by JyotiRani**
Signed-off-by: Jyoti Rani <jyotirani10@in.ibm.com>

### CODE REVIEW

**Comment by JyotiRani** [comment]
@seansund @triceam please review.

**Comment by JyotiRani** [comment]
@seansund @triceam  please review this.

**Comment by JyotiRani** [comment]
Please review and approve.

File: charts/ibm-es-instance/templates/es-instance.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+apiVersion: eventstreams.ibm.com/v1beta1
+kind: EventStreams
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace}}
```
Comment (seansund) [inline charts/ibm-es-instance/templates/es-instance.yaml]: The namespace shouldn't be provided in the values file. It should be retrieved from `{{ .Release.Namespace }}'

File: charts/ibm-es-instance/values.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Default values for eventstreams instance
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+name: es-instance
+namespace: openshift-operators
```
Comment (seansund) [inline charts/ibm-es-instance/values.yaml]: Remove this from values.yaml and pass the namespace in with the release

**Review by seansund — CHANGES_REQUESTED** [review]
Update handling of the namespace in the helm chart

**Comment by JyotiRani** [comment]
Completed the review comments https://github.com/cloud-native-toolkit/terraform-gitops-cp-event-streams-instance/issues/4 however unable to test due to common services issue on cluster. Please review the changes and confirm.

**Review by seansund — APPROVED** [review]
lgtm

**Comment by JyotiRani** [comment]
Merged with main branch

**Pull request merged**